api/src/LinkyLink/LinkyLink.csproj

```diff
@@ -27,7 +27,11 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="OpenGraph-Net" Version="3.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Cors" Version="2.2.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0-rc4" />
   </ItemGroup>
 
-
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
What is 1591 - We should consider adding a comment in here describing turning off this warning.

Reply:
Added Misc. Notes section in README.md and provided notes.